### ISSUE TITLE
typo or unclear formulation in "Internationalization and Multilingual Badges"

### ISSUE THREAD

**Issue by mgylling** [openbadges/openbadges-specification#105]
In the 2.0 spec, section [Internationalization and Multilingual Badges](http://openbadgespec.org/#Internationalization),  the third bullet says "In development". 

What does this mean? Its the only occurrence of this expression in the spec.

**Comment by ottonomy** [comment]
Good question.

This section describes the two different approaches to string internationalization in JSON-LD (see [documentation](http://json-ld.org/spec/latest/json-ld/#string-internationalization)). The first is to declare a language a document is published in. The second is to provide language declarations on individual string values in that document.

We outlined [examples](http://openbadgespec.org/examples/#Internationalization) of the first, but not the second. The example shows how to declare that a badge object is published in a certain language and how to identify other published versions of the same badge. It uses the same method to identify related versions as the BadgeClass versioning system.

That looks like

```jsonld
{
  "@context": "https://w3id.org/openbadges/v2",
  "@language": "en-us",
  "id": "https://example.org/robotics-badge.json",
  "name": "Awesome Robotics Badge",
  "related": [{
      "id": "https://example.org/beths-robotics-badge-es.json?l=es",
      "@language": "es"
    }]
}
```
We're saying: Here's a BadgeClass in US English. There is another version published in Spanish over here.

The second would look like this:
```jsonld
{
  "@context": "https://w3id.org/openbadges/v2",
  "id": "https://example.org/robotics-badge.json",
  "name": [
    {
      "@language": "en-us",
      "@value": "Awesome Robotics Badge"
    },
    {
      "@language": "es",
      "@value": "Insignia Robótica Impresionante"
    }
  ]
}
```
We'd be saying here: There are multiple languages available in this file. Process the file and filter each property value for the value of the language you would like to view the document in.

The note "In development" means that it is up to future specification groups to decide whether to support this implementation pattern. Nobody expressed a request that it be fully fleshed out in the 2.0 recommendation, and it seemed like the desired use cases could be all served by the first implementation pattern.

**Comment by mtmercury** [comment]
My concern with the second approach in the examples shown is that the payload could be quite large. Especially if there is a lot of metadata and many different localized versions of the badge.

**Comment by timothyfcook** [comment]
We agree to go with the first, more concise version for the following reasons:

- simplifies the associated validator updates
- easier to add functionality later on than remove it
- having multiple-language badges is an extreme edge-case in the current eco-system
- including multiple badge-instance with different languages within the badgeClass creates an overly large payload
- could support the same functionality for a badgeClass being published under multiple languages but publishing separate badgeClass objects for each language and introducing some kind of `sameAs` property that links the badgeClasses together.

**Comment by mgylling** [comment]
Addressed in #126.
